endpoints/kubernetes/grpc-bookstore.yaml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: esp-grpc-bookstore
+spec:
+  ports:
+  # Port that accepts JSON/HTTP requests over HTTP.
+  - port: 80
+    protocol: TCP
+    name: http
+  # Port that accepts gRPC and JSON/HTTP2 requests over HTTP.
+  - port: 9000
+    protocol: TCP
+    name: http2
+  selector:
+    app: esp-grpc-bookstore
+  type: LoadBalancer
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: esp-grpc-bookstore
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: esp-grpc-bookstore
+    spec:
+      containers:
+      - name: esp
+        image: b.gcr.io/endpoints/endpoints-runtime:latest
+        args: [
+          "-p", "80",
+          "-P", "9000",
+          "-s", "SERVICE_NAME",
+          "-v", "SERVICE_VERSION",
+          "-a", "grpc://localhost:8000"
+        ]
+        ports:
+          - containerPort: 80
+          - containerPort: 9000
+      - name: bookstore
+        image: gcr.io/endpointsv2/python-grpc-bookstore-server:latest
```

Comment:
how about we make that version consistent with the grpc python version we used to build this docker image? :1.0.1